etl/pipelines/embedding_pipeline.py

```diff
@@ -20,4 +21,5 @@ def create_embedding_store(self, documents: tuple[Document, ...]) -> VectorStore
         return FAISS.from_documents(
             documents=list(documents),
             embedding=self._create_embedding_model(),
+            distance_strategy=DistanceStrategy.COSINE,
```

Comment:
An explanatory comment would be appropriate here.

Reply:
Why are you using the cosine distance strategy instead of the default?

Reply:
I don't know why cosine similarity is the objectively better distance strategy, so I made it a parameter of `create_embedding_store`.